ui/admin/tests/acceptance/aliases/selectors.js

```diff
@@ -5,3 +5,8 @@
 
 export const FIELD_DESTINATION_ID = '[name="destination_id"]';
 export const FIELD_HOST_ID = '[name="authorize_session_arguments"]';
+
+export const CLEAR_DROPDOWN_SELECTOR =
```

Comment:
Can we delete the `_SELECTOR` part on these 3 new selectors? The constants are defined within a file call `selectors`, also imported on test files as `selectors.[CONST]` therefore feels redundant.